code/modules/antagonists/incursion/incursion.dm

```diff
@@ -91,9 +91,18 @@
 	log_admin("[key_name(admin)] made [key_name(new_owner)] and [key_name(new_owner.current)] into incursion traitor team.")
 
 /datum/antagonist/incursion/proc/equip(var/silent = FALSE)
-	var/obj/item/uplink/incursion/uplink = new(owner, owner.key, 15)
-	owner.current.equip_to_slot(uplink, ITEM_SLOT_BACKPACK)
-	to_chat(owner.current, "<span class='notice'><b>You have been equipped with a syndicate uplink located in your backpack. Activate the transponder in hand to access the market.</b></span>")
+	var/obj/item/uplink/incursion/uplink = new(owner.current.loc, owner.key, 15)
+	var/where
+	if(ishuman(owner.current))		//if he's not a human, uplink will spawn under his feet
+		var/mob/living/carbon/human/H = owner.current
+		var/static/list/slots = list(
```

Comment:
pretty sure static lists should be a typecache but idk

Reply:
I don't know, it was suggested by @TheFakeElon

Reply:
Typecaches need to be typecaches, static just means all variable references for it points to the same, single var. They only need to be created once and then they remain the whole round, saves you the processing time of making and/or storing one for every datum or reference